com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
+
+  "app-id": "com.openmusicplayer.OMP",
+
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+
+  "sdk": "org.gnome.Sdk",
+
+
+  "finish-args":
+  [
+
+    "--socket=x11",
+    "--share=network",
+    "--filesystem=host",
+    "--socket=pulseaudio"
+
+  ],
+
+  "build-options" :
+  { 
+
+    "cxxflags": "-g -std=c++17 -lstdc++fs"
```

Comment:
I'd say that `-std=c++17` should be set by your build system and `-lstdc++fs` detected by it.